docs/docs-content/clusters/edge/networking/vxlan-overlay.md

```diff
@@ -125,7 +134,7 @@ You will not be able to change the network overlay configurations after the clus
 
 3. Select the host cluster you created to view its details page.
 
-4. Select the **Nodes** tab, in the **Overlay IP Address** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration. 
+4. Select the **Nodes** tab, in the **Private Ips** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration. 
```

Comment:
This is a run-on sentence. Is the following intended?

Select the **Nodes** tab. For each host, the **Private Ips** column lists an overlay IP address within the CIDR range you provided during cluster configuration.